pkg/api/plugins.go

```diff
@@ -38,7 +38,7 @@ func (hs *HTTPServer) GetPluginList(c *models.ReqContext) response.Response {
 	// When using access control anyone that can create a data source should be able to list all data sources installed
 	// Fallback to only letting admins list non-core plugins
 	hasAccess := accesscontrol.HasAccess(hs.AccessControl, c)
-	if !hasAccess(accesscontrol.ReqOrgAdmin, accesscontrol.EvalPermission(datasources.ActionCreate)) || c.HasRole(models.ROLE_ADMIN) {
+	if !hasAccess(accesscontrol.ReqOrgAdmin, accesscontrol.EvalPermission(datasources.ActionCreate)) {
```

Comment:
We don't need `c.HasRole(models.ROLE_ADMIN)` since we already have fallback `accesscontrol.ReqOrgAdmin` in `hasAccess()` function.

Reply:
Ok, looks like we need to return non-core plugins for admins in any case, not only when access control is not enabled.